src/commandsAndMenu.tsx

```diff
@@ -732,6 +733,31 @@ export function addCommands(
     icon: openIcon.bindprops({ stylesheet: 'menuItem' })
   });
 
+  commands.addCommand(ContextCommandIDs.openFileFromDiff, {
+    label: trans.__('Open File'),
+    execute: async _ => {
```

Comment:
Let's add a caption bringing more information on the command purpose:

```suggestion
    caption: trans.__('Open file from its diff view'),
    execute: async _ => {
```